espresso-server/build.gradle

```diff
@@ -1,7 +1,25 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.31'
+    ext.getIntegerProperty = {String propertyName, int defaultValue ->
+        Object propertyValue = project.getProperties().get(propertyName)
+        if (propertyValue == null) {
+            return defaultValue
+        } else {
+            Integer.parseInt(propertyValue.toString(), 10)
+        }
+    }
+    ext.getStringProperty = { String propertyName, String defaultValue ->
+        Object propertyValue = project.getProperties().get(propertyName)
+        if (propertyValue == null) {
+            return defaultValue
+        } else {
+            propertyValue.toString()
+        }
+    }
+
+    ext.kotlin_version = getStringProperty('kotlin_version', '1.3.31')
+    ext.agp_version = getStringProperty('agp_version', '3.4.2')
```

Comment:
what `agp` means?

Reply:
Android Gradle plugin

Reply:
I'd prefer to use full name instead of abbreviation then. It's not very common